frontend/src/views/auth/Login.vue

```diff
@@ -196,8 +197,14 @@ export default {
       this.error = null
       this.$auth
         .login(this.email, this.password)
-        .then(() => {
+        .then(async () => {
           this.$router.replace(this.$route.query.redirect || '/')
```

Comment:
replace is async. Do you not await the promise on purpose?
https://router.vuejs.org/api/interfaces/Router.html#Methods-replace

Reply:
So far, we did not want to wait, because we had nothing to do after the navigation completes. If I move the `$router.replace` last as you propose, I will again not need to await or return the promise. So, yes, it is a conscious choice to not await it, because it would be pointless.